### PR TITLE
CP-41366: Rename Citrix Hypervisor to XenServer

### PR DESCRIPTION
**Pull request by rosslagerwall**
Signed-off-by: Ross Lagerwall <ross.lagerwall@citrix.com>

---

This doesn't anything regarding the SDK.

### CODE REVIEW

**Review by psafont — APPROVED** [review]

**Comment by kc284** [comment]
Don't worry about ocaml/sdk-gen, it will be handled by the UI team.

**Review by edwintorok — COMMENTED** [review]
I think we're not supposed to have branding in the code, but load the branding from /etc/xensource-inventory instead. Obviously that doesn't work for documentation,
and you'd first need to load the inventory to know the brand, and if that is missing...
But if that is missing something has gone really bad with the install, perhaps we could have a more neutral message: "unable to load XAPI inventory configuration?"

File: scripts/xe-xentrace [hunk 1]
```diff
@@ -31,7 +31,7 @@ done
 SIZE=$((${SIZE_GB} * 1024 * 1024 * 1024))
 
 if [ ! -e @INVENTORY@ ]; then
-  echo Must run on a Citrix Hypervisor host.
+  echo Must run on a XenServer host.
```
[the remark this answers is not in the document]
Comment (rosslagerwall) [reply scripts/xe-xentrace:34]: OK, I've updated the `scripts/xe-*` files. I think the rest are docs and examples so not appropriate to get branding from `/etc/xensource-inventory` for them.

[the remark this answers is not in the document]
Comment (edwintorok) [reply scripts/xe-xentrace:34]: Did you push the commit, I don't see it on github

**Review by lindig — APPROVED** [review]

**Comment by lindig** [comment]
This is changing user-visible documentation but not copyright - which is the right approach.

File: scripts/xe-xentrace (hunk 1 above)
[the remark this answers is not in the document]
Comment (rosslagerwall) [reply scripts/xe-xentrace]: Oops... I have now pushed it to the correct place.

**Review by edwintorok — APPROVED** [review]

**Pull request merged**